src/Bridge/Doctrine/EventListener/PublishMercureUpdatesListener.php

```diff
@@ -47,8 +48,10 @@ final class PublishMercureUpdatesListener
     private $createdEntities;
     private $updatedEntities;
     private $deletedEntities;
+    private $requestStack;
```

Comment:
We shouldn’t use request stack here, because it makes the feature not fully compliant with GraphQL, and create a hard dependency to Symfony (we want to support Laravel and maybe PSR-7 at some point).

Reply:
The only solution i see is making use of $GLOBALS['request']... unless api-platform has some other way to get the current request

Reply:
btw, i thing Laravel also uses Symfony's HttpFoundation...

Reply:
Yes we have our own abstraction layer: https://github.com/api-platform/core/blob/master/src/EventListener/ReadListener.php#L64-L69

However I'm not sure it's the good way to handle this problem. I would only make the format configurable for now.

Yes Laravel uses HttpFoundation, but it doesn't use the `RequestStack` class.